DXRFixes/DeusEx/Classes/Lamp2.uc

```diff
@@ -0,0 +1,6 @@
+class DXRLamp2 injects Lamp2;
```

Comment:
If this change makes sense, it should just go into the DXRLamp2 file that I made yesterday, in the DXRando folder - not sure that the light radius needs to be explicitly separated if we were to go and replace the lamps in other mods